app/api/helpers/permission_manager.py

```diff
@@ -122,6 +122,13 @@ def is_user_itself(view, view_args, view_kwargs, *args, **kwargs):
     Otherwise the user can only access his/her resource.
     """
     user = current_user
+    if kwargs['source'] and kwargs['source'] == 'users':
+        if not user.is_admin and not user.is_super_admin and user.id != kwargs['user_id']:
+            view_kwargs['schema']= 'public'
+        else:
+            view_kwargs['schema']='private'
+        return view(*view_args, **view_kwargs)
```

Comment:
Not correct, see the implementation in other parts of the server

Reply:
I understand, but in other parts of the server the API view is written differently checks are done inside each kwargs, but the user view has a global check of is_user_itself which needs to be carried out, if not we have to implement a jwt_required in before_get to fetch current_user followed by an admin check which will still miss is_user_itself check to provide private data.

And if we by default do is_user_itself check, we'll be unable to serve data to random users, limited one as the Access Denied message shows up.

My soluton is to do an additional check in is_user_itself for user view and behave differently

Reply:
Ok my changes failed circleci this should be wrong